sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/DeclarativeConfiguration.java

```diff
@@ -82,7 +83,7 @@ private DeclarativeConfiguration() {}
    *
    * @throws DeclarativeConfigException if unable to parse or interpret
    */
-  public static OpenTelemetrySdk parseAndCreate(InputStream inputStream) {
+  public static ExtendedOpenTelemetrySdk parseAndCreate(InputStream inputStream) {
```

Comment:
This is cool - frees up callers from needing to worry about initializing `SdkConfigProvider`